lib/linguist/heuristics.yml

```diff
@@ -216,6 +216,8 @@ disambiguations:
     pattern: '^<\?(?:php)?'
   - language: SourcePawn
     pattern: '^public\s+(?:SharedPlugin(?:\s+|:)__pl_\w+\s*=(?:\s*{)?|(?:void\s+)?__pl_\w+_SetNTVOptional\(\)(?:\s*{)?)'
+  - language: NASL
+    named_pattern: incNASL
```

Comment:
We don't need a named pattern here since these heuristic rules are only used here.

Reply:
Updated in latest commit.